### ISSUE TITLE
Some sort of Parsing error..

### ISSUE THREAD

**Issue by YukiRainee** [mangosango/clive#19]
`[1609026142083] INFO  (TwitchJS/Chat/14212 on raspberrypi): PRIVMSG/#brettdoesgaming rainee4357: https://clips.twitch.tv/MagnificentCleverCaribouKappaWealth
[1609026142110] ERROR (TwitchJS/Chat/Client/14212 on raspberrypi): Parsing error encountered. Please create an issue: https://github.com/twitch-js/twitch-js/issues/new?title=Parsing%20error%20encountered&body=%40badge-info%3D%3Bbadges%3Dmoderator%2F1%2Cglitchcon2020%2F1%3Bclient-nonce%3Df8bef847ffb4ce7fa442616102795c8c%3Bcolor%3D%23952197%3Bdisplay-name%3Drainee4357%3Bemotes%3D%3Bflags%3D%3Bid%3D9e82720a-f4a9-4548-9690-2093135ae218%3Bmod%3D1%3Broom-id%3D49651280%3Bsubscriber%3D0%3Btmi-sent-ts%3D1609026141961%3Bturbo%3D0%3Buser-id%3D49262317%3Buser-type%3Dmod%20%3Arainee4357%21rainee4357%40rainee4357.tmi.twitch.tv%20PRIVMSG%20%23brettdoesgaming%20%3Ahttps%3A%2F%2Fclips.twitch.tv%2FMagnificentCleverCaribouKappaWealth%0D%0A {}
[1609026142114] INFO  (TwitchJS/Chat/14212 on raspberrypi): PARSE_ERROR_ENCOUNTERED : [TwitchJS] Parse error encountered [Chat]
/home/pi/Clive/node_modules/twitch-js/lib/Chat/Client.js:210
            throw errorMessage;
            ^

Error: [TwitchJS] Parse error encountered [Chat]
    at ParseError.ChatError [as constructor] (/home/pi/Clive/node_modules/twitch-js/lib/Chat/Errors.js:26:24)
    at new ParseError (/home/pi/Clive/node_modules/twitch-js/lib/Chat/Errors.js:50:28)
    at Client._handleMessage (/home/pi/Clive/node_modules/twitch-js/lib/Chat/Client.js:207:32)
    at WebSocket.onMessage (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/event-target.js:120:16)
    at WebSocket.emit (events.js:198:13)
    at Receiver.receiverOnMessage (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/websocket.js:800:20)
    at Receiver.emit (events.js:198:13)
    at Receiver.dataMessage (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/receiver.js:436:14)
    at Receiver.getData (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/receiver.js:366:17)
    at Receiver.startLoop (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/receiver.js:142:22)
    at Receiver._write (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/receiver.js:77:10)
    at doWrite (_stream_writable.js:415:12)
    at writeOrBuffer (_stream_writable.js:399:5)
    at Receiver.Writable.write (_stream_writable.js:299:11)
    at TLSSocket.socketOnData (/home/pi/Clive/node_modules/twitch-js/node_modules/ws/lib/websocket.js:875:35)
    at TLSSocket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:269:11)
    at TLSSocket.Readable.push (_stream_readable.js:224:10)
    at TLSWrap.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
[nodemon] app crashed - waiting for file changes before starting...`

Seem to be getting some sort of error when I try and post a clip into chat not sure if its an issue with clive or twitch.js... I'm assuming it might be twitch.js here but I thought I post it here just in case

**Comment by JonasDemand** [comment]
[1612448089223] INFO  (TwitchJS/Chat/31564 on Jonas-PC): PRIVMSG/#monxyow Kedini: https://www.twitch.tv/monxyow/clip/DeadTrustworthyEggDBstyle
[1612448089225] ERROR (TwitchJS/Chat/Client/31564 on Jonas-PC): Parsing error encountered. Please create an issue: https://github.com/twitch-js/twitch-js/issues/new?title=Parsing%20error%20encountered&body=%40badge-info%3Dfounder%2F1%3Bbadges%3Dmoderator%2F1%2Cfounder%2F0%3Bclient-nonce%3D006b2d0826a5721af46ba740a127cf55%3Bcolor%3D%231E90FF%3Bdisplay-name%3DKedini%3Bemotes%3D%3Bflags%3D%3Bid%3D7760947e-3f1b-44bf-abd0-4adda70c1848%3Bmod%3D1%3Broom-id%3D180354836%3Bsubscriber%3D0%3Btmi-sent-ts%3D1612448089564%3Bturbo%3D0%3Buser-id%3D172936253%3Buser-type%3Dmod%20%3Akedini%21kedini%40kedini.tmi.twitch.tv%20PRIVMSG%20%23monxyow%20%3Ahttps%3A%2F%2Fwww.twitch.tv%2Fmonxyow%2Fclip%2FDeadTrustworthyEggDBstyle%0D%0A {}
[1612448089226] INFO  (TwitchJS/Chat/31564 on Jonas-PC): PARSE_ERROR_ENCOUNTERED : [TwitchJS] Parse error encountered [Chat]
C:\Git\cliveP\node_modules\twitch-js\lib\Chat\Client.js:210
            throw errorMessage;
            ^

ParseError: [TwitchJS] Parse error encountered [Chat]
    at ParseError.ChatError [as constructor] (C:\Git\cliveP\node_modules\twitch-js\lib\Chat\Errors.js:26:24)
    at new ParseError (C:\Git\cliveP\node_modules\twitch-js\lib\Chat\Errors.js:50:28)
    at Client._handleMessage (C:\Git\cliveP\node_modules\twitch-js\lib\Chat\Client.js:207:32)
    at WebSocket.onMessage (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\event-target.js:120:16)
    at WebSocket.emit (events.js:315:20)
    at Receiver.receiverOnMessage (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\websocket.js:800:20)
    at Receiver.emit (events.js:315:20)
    at Receiver.dataMessage (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\receiver.js:436:14)
    at Receiver.getData (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\receiver.js:366:17)
    at Receiver.startLoop (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\receiver.js:142:22)
    at Receiver._write (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\receiver.js:77:10)
    at writeOrBuffer (internal/streams/writable.js:358:12)
    at Receiver.Writable.write (internal/streams/writable.js:303:10)
    at TLSSocket.socketOnData (C:\Git\cliveP\node_modules\twitch-js\node_modules\ws\lib\websocket.js:875:35)
    at TLSSocket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at TLSSocket.Readable.push (internal/streams/readable.js:223:10)
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23) {
  timestamp: 2021-02-04T14:14:49.226Z,
  _raw: '@badge-info=founder/1;badges=moderator/1,founder/0;client-nonce=006b2d0826a5721af46ba740a127cf55;color=#1E90FF;display-name=Kedini;emotes=;flags=;id=7760947e-3f1b-44bf-abd0-4adda70c1848;mod=1;room-id=180354836;subscriber=0;tmi-sent-ts=1612448089564;turbo=0;user-id=172936253;user-type=mod :kedini!kedini@kedini.tmi.twitch.tv PRIVMSG #monxyow :https://www.twitch.tv/monxyow/clip/DeadTrustworthyEggDBstyle\r\n',
  command: 'PARSE_ERROR_ENCOUNTERED'
}
[nodemon] app crashed - waiting for file changes before starting...


Having the same issue right now, do you have a solution to this problem?

**Comment by jaredcat** [comment]
@YukiRainee @JonasDemand there were some changes on Twitch's side that broke this project. A lot of changes and fixes were just merged, so try it again!

Feel free to open a new issue if you run into any problems
